untitled/src/HomeWorkApp.java

```diff
@@ -0,0 +1,48 @@
+
+public class HomeWorkApp {
+    public static void main(String[] args) {
+        printThreeWords();
+        checkSumSign();
+        printColor();
+        compareNumbers();
+    }
+
+    public static void printThreeWords() {
+        System.out.println("Orange");
+        System.out.println("Banana");
+        System.out.println("Apple");
+    }
+    public static void checkSumSign() {
+        int a=2;
+        int b=3;
+        if (a+b>=0) {
+            System.out.println("Сумма положительная");
+        }
+        else {
+            System.out.println("Сумма отрицательная");
+        }
+    }
+    public static void printColor(){
+        int value = 100;
+        if (value <= 0) {
+            System.out.println("Красный");
+        }
+        else if ((value >0) && (value<=100)) {
```

Comment:
поскольку данное условие будет проверяться в случае неудачи первого выше, то можно упростить
```suggestion
        else if ((value<=100)) {
```